Sources/HTTP/HTTPStreamingParser.swift

```diff
@@ -307,7 +307,7 @@ public class StreamingParser: HTTPResponseWriter {
     var headersWritten = false
     var isChunked = false
     
-    /// Create a `HTTPRequest` struct from the parsed information 
+    /// Create a `Request` struct from the parsed information 
```

Comment:
As this PR is only covering the module breakout this should still be HTTPRequest.